src/integrationTest/java/com/mongodb/hibernate/junit/MongoExtension.java

```diff
@@ -61,22 +69,30 @@ public void beforeAll(ExtensionContext context) throws Exception {
         }
     }
 
+    @Override
+    public void afterAll(ExtensionContext context) {
+        disableFailPoint();
+        STATE.mongoDatabase().drop();
```

Comment:
The data should not be dropped after running a test / all tests, because having that data may help analyzing the behavior. Unlike enabled failpoints, such data is harmless, given that we delete it before each test.

```suggestion
```

P.S. If it were actually important to delete the data in `afterAll`, we should have used `finally` to make sure that an exception from `disableFailPoint` does not prevent that from happening.